plugins/modules/openssl_pkcs12.py

```diff
@@ -51,6 +51,13 @@
     type: list
     elements: path
     aliases: [ca_certificates]
+  other_certificates_content:
+    description:
+      - List of other certificates to include.
```

Comment:
I think this needs similar explanations as `other_certificates` for what the content is and about `other_certificates_parse_all`.